controller.go

```diff
@@ -1056,3 +1058,24 @@ func convertConstraintMatchesAny(source interface{}) map[string][]interface{} {
 	}
 	return result
 }
+
+// Tags implements Controller.
+func (c *controller) Tags() ([]Tag, error) {
+	params := NewURLParams()
+	source, err := c.getQuery("tags", params.Values)
```

Comment:
It seems a little odd that the way to do the controller tags to pass an empty url.Values, but I trust you to have studied the MAAS api itself for this.

Reply:
You are right - this should just be `nil`. Fixed and thanks!